### PR TITLE
some script improvements

### PR DESCRIPTION
**Pull request by acheshkov**
minor changes

### CODE REVIEW

**Comment by acheshkov** [comment]
@yegor256 please review

**Comment by 0crat** [comment]
This pull request [#43](https://github.com/yegor256/aibolit/issues/43) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/40b2153f-d992-4d64-87eb-f39fa3d0741e); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by carlosmiranda** [comment]
@acheshkov what task (Github ticket) is this change referring to, and specifically what improvements are we trying to make?

**Comment by carlosmiranda** [comment]
@acheshkov ping

**Comment by acheshkov** [comment]
@carlosmiranda, there were no ticket. This commit moves some file downloading and unzipping form python script to Makefile. Sorry for delayed reply.

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

File: requirements.txt [hunk 1]
```diff
@@ -5,4 +5,5 @@ javalang==0.12.0
 sphinx==2.3.1
 beautifulsoup4==4.8.2
 numpy==1.18.1
+pandas==1.0.0
```
Comment (carlosmiranda) [inline requirements.txt]: @acheshkov what does this have to do with the rest of the changes?

**Comment by yegor256** [comment]
@carlosmiranda I decided to merge, since it's urgent

**Comment by 0crat** [comment]
Order was finished: +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)
